.eslintrc

```diff
@@ -81,6 +81,7 @@
     "react/jsx-curly-spacing": ["error", "never"],
     "react/jsx-handler-names": ["error"],
     "react/prefer-es6-class": ["off"],
+    "react/prefer-stateless-function": ["off"],
```

Comment:
This makes moving from a stateless component, to complex component, more difficult. Same rational used in requiring curlies for one line conditionals and enforcing not having them for one line arrow functions.